templates/dashboard/_upcoming_reminders.html.twig

```diff
@@ -0,0 +1,28 @@
+{% trans_default_domain 'reminder' %}
+
+<div class="rounded-sm ">
+    <h2 class="h3">{% trans %}Upcomming reminders{% endtrans %}</h2>
+    {% if upcoming_reminders|length > 0 %}
+        {% for key, value in upcoming_reminders %}
```

Comment:
Consider renaming `key` to `date` and `value` to `reminders`.

Reply:
Renamed to `date` and `reminders`.